_data/roles.yml

```diff
@@ -0,0 +1,20 @@
+- name: Registry Metadata Steward
+  commitment: 2-4 hours per month
+  requirements:
+    - Love for metadata
+    - basic python
+    - community mindset
+  description: |
+    You are responsible for shepherding our registry
+    metadata according to our principles and SOPs, merge changes and generally
+    protect and guard them. This involves also facilitating the implementation of QC
+    checks.
+- name: OBO NOR Dashboard maintainer
+  commitment: 2-6 hours per month
+  requirements:
+    - Basic knowledge of dashboard
+    - Social skills are a nice to have
+    - Infinite patience also helpful
+  description: |
```

Comment:
you could always make this better with more structured information, like a list of the current people who have the role

Reply:
I separated these into open opportunities and currently filled roles. Let me know what you think and please suggest any other changes, thanks!

Reply:
i'd say better to have the fact that the role is looking for new people to be a property of the role itself and not the way that roles are grouped together (so maybe add a new key `open: true` or `open: false`)

Reply:
I revised this